packages/apps/docs/src/docs/kadena/overview.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Overview of Kadena
 description: Kadena makes blockchain work for everyone.
-menu: Kadena
+menu: Overview
```

Comment:
I changed this Kadena menu to Overview, as far as i could see for other top level menu items, it was using the label for the menu.